test/gtest/uct/ib/test_rc.cc

```diff
@@ -144,17 +147,34 @@ class test_rc_get_limit : public test_rc {
         m_max_get_zcopy = 4096;
         modify_config("RC_MAX_GET_ZCOPY",
                       ucs::to_string(m_max_get_zcopy).c_str());
-        modify_config("RC_TX_CQ_LEN", "32");
+
+        modify_config("RC_TX_QUEUE_LEN", "32");
 
         m_comp.count = 300000; // some big value to avoid func invocation
         m_comp.func  = NULL;
     }
 
     void init() {
+#if ENABLE_STATS
+        stats_activate();
+#endif
         test_rc::init();
         check_skip_test();
     }
 
+#if ENABLE_STATS
```

Comment:
do we really need this '#if' ?

Reply:
in theory we could build tests without stats

Reply:
would it really fail to compile in this case? seems the code here would just do nothing if no stats

Reply:
i'll check, but imo better have it in any case as "documentation"

Reply:
it fails with
```
common/test.cc:104: Failure
Failed
Invalid UCS configuration for STATS_DEST : file:/dev/null, error message: No such element(-12)
common/test.cc:104: Failure
Failed
Invalid UCS configuration for STATS_TRIGGER : exit, error message: No such element(-12)
uct/uct_test.cc:581: Failure
Value of: ucs_stats_is_active()
  Actual: false
Expected: true
[jazz05:125628:0:125628]       test.cc:246  Assertion `m_state == FINISHED || m_state == SKIPPED || m_state == ABORTED' failed: state=1

```